Assets/MixedRealityToolkit/Inspectors/Profiles/MixedRealityInputSystemProfileInspector.cs

```diff
@@ -69,131 +69,119 @@ protected override void OnEnable()
             controllerVisualizationProfile = serializedObject.FindProperty("controllerVisualizationProfile");
             handTrackingProfile = serializedObject.FindProperty("handTrackingProfile");
 
-            providerFoldouts = new bool[dataProviderConfigurations.arraySize];
+            if (providerFoldouts == null || providerFoldouts.Length != dataProviderConfigurations.arraySize)
+            {
+                providerFoldouts = new bool[dataProviderConfigurations.arraySize];
+            }
         }
 
         public override void OnInspectorGUI()
         {
-            RenderTitleDescriptionAndLogo(
-                "Input System Profile",
-                "The Input System Profile helps developers configure input for cross-platform applications.");
-
-            if (!MixedRealityInspectorUtility.CheckMixedRealityConfigured(true, !RenderAsSubProfile))
+            if (!RenderProfileHeader(ProfileTitle, string.Empty))
             {
                 return;
             }
 
-            if (DrawBacktrackProfileButton("Back to Configuration Profile", MixedRealityToolkit.Instance.ActiveProfile))
+            // TODO: Troy
+            /*
+            if (!MixedRealityInspectorUtility.CheckMixedRealityConfigured(true, !RenderAsSubProfile))
             {
                 return;
```

Comment:
Do you know if this was supposed to be uncommented (or deleted?)

Reply:
This comment is on an outdated version of the file. It has already been removed